lib/shopify-cli/helpers/partners_api.rb

```diff
@@ -44,7 +44,10 @@ def authenticated_req(ctx)
           Tasks::AuthenticateIdentity.call(ctx)
           retry
         rescue API::APIRequestNotFoundError
-          ctx.puts("{{x}} error: Your account was not found. Please sign up at https://partners.shopify.com/signup")
+          raise(
```

Comment:
please use `ctx.abort()`

Reply:
I think thats only in your feature branch. In master its only `ctx.error` thats available but I didn't want to cause any conflicts when you merged. I can rebase off the feature branch if you want? Or use `ctx.error`?

Reply:
User ctx.error then please.